components/web/nginx/Makefile

```diff
@@ -13,63 +13,62 @@
 #
 
 BUILD_BITS= 64
-# Alas nginx ignores the openssl settings
-OPENSSL_VERSION= 3.1
-USE_PARALLEL_BUILD= yes
```

Comment:
Why this got removed?  Does it fail to build when the parallel build is enabled?